.github/workflows/router-docker-publish.yml

```diff
@@ -46,15 +47,21 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Build image
-        run: docker build . --file router.Dockerfile --tag $IMAGE_NAME
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v2
 
-      - name: Log into registry
-        run: docker login -u ${{ secrets.DOCKER_USER }} -p ${{ secrets.DOCKER_PASSWORD }}
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v2
 
-      - name: Push image
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: export version variable
         run: |
-          IMAGE_ID=kserve/$IMAGE_NAME
+          IMAGE_ID=$DOCKER_USER/$IMAGE_NAME
```

Comment:
so the run finished successfully, but where did it push now? https://github.com/kserve/kserve/actions/runs/3759474542/jobs/6389121367#step:7:16

in any case not where I had expected it: https://hub.docker.com/r/kserve/kserve-controller/tags 😅

Reply:
wanted to comment on `kserve-controlling` PR, but also valid for this PR actually

Reply:
cc @yuzisun

Reply:
opened #2619